product/scanning.py

```diff
@@ -94,6 +94,14 @@ def retrieve_scan_result(
         product = self._api.scan_result(product_id=product_id)
         return product
 
+    @lru_cache(maxsize=8)
```

Comment:
why 8?

Reply:
Seemed like a good idea to prevent a large cache at the time. I realise now that this is currently not really possible and thus removed the limit.